plugin/restore/mysql/mysql.go

```diff
@@ -12,25 +12,25 @@ import (
 
 	"github.com/bytebase/bytebase/api"
 	mysql "github.com/bytebase/bytebase/plugin/db/mysql"
+	"github.com/bytebase/bytebase/resources/mysqlbinlog"
 )
 
-// TODO(zp): refactor this when sure the mysqlbinlog path
-var mysqlbinlogBinPath = "UNKNOWN"
-
 const (
 	// MaxDatabaseNameLength is the allowed max database name length in MySQL
 	MaxDatabaseNameLength = 64
 )
 
 // Restore implements recovery functions for MySQL
 type Restore struct {
-	driver *mysql.Driver
+	driver         *mysql.Driver
+	mysqlbinlogIns *mysqlbinlog.Instance
```

Comment:
```suggestion
	mysqlbinlog *mysqlbinlog.Instance
```
It's two-fold.
1. The "Ins" is not meaningful.
2. IMO it's ok for a field to have the same name as a package name.